validator/remote/src/main/java/tech/pegasys/teku/validator/remote/apiclient/OkHttpValidatorRestApiClient.java

```diff
@@ -131,29 +128,6 @@ public Optional<List<ValidatorResponse>> getValidators(final List<String> valida
         .map(response -> response.data);
   }
 
-  /**
-   * <a
-   * href="https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Beacon/postStateValidators">POST
-   * Get validators from state</a>
-   */
-  @Override
-  public Optional<List<ValidatorResponse>> postValidators(final List<String> validatorIds) {
-    final PostStateValidatorsRequest requestBody = new PostStateValidatorsRequest(validatorIds);
-    return post(
-            GET_VALIDATORS,
-            EMPTY_MAP,
-            requestBody,
-            createHandler(GetStateValidatorsResponse.class)
-                .withHandler(
-                    (request, response) -> {
-                      throw new PostStateValidatorsNotExistingException();
-                    },
-                    SC_BAD_REQUEST,
-                    SC_NOT_FOUND,
-                    SC_METHOD_NOT_ALLOWED))
```

Comment:
this is basically why the acceptance test is failing. we're not implementing this fallback, so we loop on getting 404 for ever.